proposed/http-middleware/middleware-meta.md

```diff
@@ -1,49 +1,56 @@
-PSR-15 Meta Document
-====================
+HTTP Server Middleware
+======================
 
 1. Summary
 ----------
 
 The purpose of this PSR is to provide an interface that defines the formal
-method signature for HTTP Middleware that is compatible with HTTP Messages,
-as defined in PSR-7.
+method signature for HTTP Server Middleware that is compatible with HTTP
+Messages, as defined in [PSR-7][psr7].
+
+[psr7]: http://www.php-fig.org/psr/psr-7/
 
 2. Why Bother?
 --------------
 
-The general concept of reusable middleware was popularized by [StackPHP][stackphp].
-Since the release of the HTTP Messages standard, a number of frameworks have
-adopted middleware that uses HTTP Message interfaces.
+The HTTP Messages specification does not contain any reference to HTTP Middleware.
+
+The design pattern used by middleware has existed for many years as the
+[pipeline pattern][pipeline], or more specifically, "linear pipeline processing".
```

Comment:
> > […] nor does they know how it was possible that this sentence got here.
>
> Like everything in this PSR, it was either written by me, or added by me as editor.
> …
> I fail to see why you are objecting so strongly to it.

WHY is the important question, and WHO is only important during reviewing – Your are unwilling to understand that, aren't you?

---

> middleware processing is without question an example

:confused: I've never, claimed the opposite! Please, reread my comment above – and btw, I believe you just try to distract from the real issue, because you draw attention to one single argument I've brought above and ignore all the others.